packages/amplication-client/src/Resource/git/ServiceConfigurationGitSettings.tsx

```diff
@@ -99,7 +99,7 @@ const ServiceConfigurationGitSettings: React.FC<Props> = ({
     [resource.id, trackEvent, updateResourceOverrideStatus]
   );
 
-  const isToggleDisable = currentWorkspace?.gitOrganizations?.length === 0;
+  const isToggleDisable = currentWorkspace?.gitOrganizations?.length !== 0;
```

Comment:
Thanks @Jyotiraditya24 for the PR. 
Can you please confirm that the original functionality still works?

If you have a git provider connected already at the project level, can you override it at the resource level?